src/errors.rs

```diff
@@ -39,16 +34,16 @@ pub enum Error<E>
     ///
     /// Typically this indicates a faulty implementation of `serde::Serialize`.
     ServerSerialize(String),
-    /// The server canceled the response before it was completed.
-    ReplyCanceled,
     /// The server was unable to reply to the rpc for some reason.
+    ///
+    /// This is a service-specific error. It is individually specified in the
+    /// `service!` macro for each rpc.
```

Comment:
It is returned in the implementation, but the type is specified in the macro invocation. How about, "The type is individually specified in the `service!` macro for each rpc."